lib/streamlit/runtime/caching/cache_resource_api.py

```diff
@@ -275,7 +275,7 @@ def __call__(
         )
 
     @staticmethod
-    @gather_metrics("clear_singleton")
+    @gather_metrics("clear_resource_caches")
```

Comment:
maybe we could use `clear_recource_cache`, WDYT?

Reply:
I used "caches" plural here because this function clears _all_ resource caches (as opposed to the cache-specific clear function, which just clears a single cache)